torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19491,6 +19491,27 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            supports_gradgrad=True,
            supports_out=False,
            ),
+    OpInfo('unbind_copy',
+           dtypes=all_types_and_complex_and(torch.complex32, torch.bool, torch.float16, torch.bfloat16),
+           ref=reference_unbind,
+           sample_inputs_func=sample_inputs_unbind,
+           error_inputs_func=error_inputs_unbind,
+           supports_forward_ad=True,
+           supports_fwgrad_bwgrad=True,
+           supports_gradgrad=True,
+           supports_out=True,
+           check_batched_grad=False,
+           skips=(
+               # grad actually does work with out=
+               DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_out_requires_grad_error'),
```

Comment:
Is there any other example where requires_grad + out= work together? I don't know of any which is making me suspicious about this.

Reply:
```
from torch.testing._internal.common_methods_invocations import op_db
a = [o for o in op_db if o.supports_out and o.supports_autograd]
print(len(a))  # prints 268
print(len([i for i in a if i.name.endswith('_copy')]))  # prints 10
```

Reply:
Those ops have an `out=` overload and support gradients, yes. I am asking if any of those ops also have this test x-failed. The test asserts that an error is raised if we are tracking gradients on the `out=` argument which is expected why is this op an exception to that rule?